RouteSourceAdapter.py

```diff
@@ -0,0 +1,77 @@
+import requests
+
+
+class RouteSource:
+    def get_routes(self): pass
+
+
+class RouteSourceAdapter(RouteSource):
+    def __init__(self, route_source: RouteSource):
+        self.adapter = route_source
+
+    def get_routes(self):
+        data = self.adapter.get_routes()
+        print(data)
+        return data
+
+class EveScoutSource(RouteSource):
+    def __init__(self):
+        self.url = 'https://www.eve-scout.com/api/wormholes'
+
+    def get_routes(self):
+        with requests.session() as s:
```

Comment:
Since you are only making one request there is no need to make a new session for this. Just use requests.get.